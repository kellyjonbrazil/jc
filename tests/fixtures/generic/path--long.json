{
  "path": "/Library/Application Support/Script Editor/Templates/Cocoa-AppleScript Applet.app/Contents/Info.plist",
  "parent": "/Library/Application Support/Script Editor/Templates/Cocoa-AppleScript Applet.app/Contents",
  "filename": "Info.plist",
  "stem": "Info",
  "extension": "plist",
  "path_list": [
    "/",
    "Library",
    "Application Support",
    "Script Editor",
    "Templates",
    "Cocoa-AppleScript Applet.app",
    "Contents",
    "Info.plist"
  ]
}