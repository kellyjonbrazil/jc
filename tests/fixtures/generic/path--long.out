/Library/Application Support/Script Editor/Templates/Cocoa-AppleScript Applet.app/Contents/Info.plist